buyer-api/test/routes/health.test.js

```diff
@@ -1,26 +0,0 @@
-import test from 'ava';
```

Comment:
I would suggest to keep this but to scope the test to only include the route. You can use the `.test.ignore.js` pattern maybe?